packages/legend-query-builder/src/components/execution-plan/SQLExecutionNodeViewer.tsx

```diff
@@ -35,14 +60,136 @@ export const SQLExecutionNodeViewer: React.FC<{
   resultColumns: SQLResultColumn[];
   executionPlanState: ExecutionPlanState;
 }> = observer((props) => {
-  const { query } = props;
+  const { query, resultColumns, executionPlanState } = props;
+
+  const stringifyDataType = (
```

Comment:
could we use stringifyDataType from V1_ExecutionPlanTransformer.ts ?

Reply:
yes please

Reply:
Migrated stringifyDataType to legend-graph 👍